guides/release/testing/index.md

```diff
@@ -1,199 +1,15 @@
-Ember gives you **the power to write tests and be productive from day one**. You can be confident that your app will be correct today and years from now. A question remains: *How* should you write tests?
+Ember gives you **the power to write tests and be productive from day one**. You can be confident that your app will be correct today and years from now. A question remains: _How_ should you write tests?
 
 Since tests are a core part of the Ember framework and your development cycle, we will dedicate several sections to learning how to write tests.
 
-In this section, we will cover why testing is important, what tools can help you with testing, and how to run and debug your tests.
-
+In this section, we will cover why testing is important and how to run, debug and filter your tests.
 
 ## Why Do I Need Tests?
 
 Writing tests is a necessary ingredient if you want to guarantee users and stakeholders that your app, whether small or large, will function as intended at any given time. The larger your app, the more costly and error-prone manual testing becomes.
 
 Writing tests is also a fun activity, a nice change of pace from delivering features daily, and a way to help you refactor code and improve as a developer. Tests can also serve as a living documentation—a key element in onboarding new developers.
```

Comment:
what is documentation-a here?

Reply:
It is not a single word, the sentence is like `documentation` : `a key element in onboarding new developers.
Maybe we can add a space to distinguish it more clearly.